### DISCUSSION TITLE
Remove imported ground markers

### DISCUSSION THREAD

**Discussion by discomethod** [runelite/runelite#16955] [Ideas]
I recently imported a slew of ground markers for the Hallowed Sepulchre, but after running a few laps, decided I didn't like using them.

There could be a toggle to instead _remove_ the inputted list of markers. This way I can just paste in the list of ground tiles again to remove them all.

**Comment by Nightfirecat** [comment]
If you export your ground markers prior to importing new ones, you can then clear markers and re-import your old ones. I don't think we'll be adding any kind of "undo" feature to ground marker importing.

**Comment by geheur** [comment]
You could use the tile marker import string to create a tile pack via the tile packs plugin hub plugin, then you can enable/disable/delete it whenever you want.
